src/rpc/ice.js

```diff
@@ -629,8 +629,14 @@ Ice.prototype._connect_tcp_active_passive_pair = function(session) {
         }
         session.tcp = net.connect(session.remote.port, session.remote.address);
         session.tcp.on('error', function(err) {
-            dbg.log0('Got error', err);
+            dbg.log0('Got error', err.message);
```

Comment:
Why not `dbg.error()`?

Reply:
because a closed ICE connection is not a real error - there are many valid cases for it and it happens in normal flow.